frontend/testing/playwright/scripts/setup.ts

```diff
@@ -0,0 +1,68 @@
+﻿import path from 'path';
+import fs from 'fs';
+import os from 'os';
+
+const giteaApi = require('../../../../development/utils/gitea-api');
+
+// Configure the dotenv to read form the .env file on root of monorepo.
+require('dotenv').config({ path: path.resolve(__dirname, '../../../../.env') });
+
+const environment: Record<string, string> = {
+  PLAYWRIGHT_TEST_BASE_URL: 'http://studio.localhost',
+  PLAYWRIGHT_DESIGNER_APP_NAME: 'auto-test-app',
+
+  PLAYWRIGHT_USER: process.env.GITEA_CYPRESS_USER,
```

Comment:
Should we maybe rename ```GITEA_CYPRESS_USER``` to something a bit more generic now that it is used for Playwright as well as Cypress?😄

Reply:
Sure, that should be done within the following issue: https://github.com/Altinn/altinn-studio/issues/11991